lib/errors.rb

```diff
@@ -51,6 +51,9 @@ class UserNotFound < NotFound
   class Blocked < Forbidden
   end
 
+  class LoginRequired < Forbidden
```

Comment:
i guess a new base error `NotAuthorized` with a 401 status would make more sense for that one